proteus/mprans/RANS2P2D.h

```diff
@@ -862,11 +862,11 @@ namespace proteus
       /* duc_du = u/(uc+1.0e-12); */
       /* duc_dv = v/(uc+1.0e-12); */
       //semi-implicit quadratic term
-      uc = sqrt(uStar*uStar+vStar*vStar*+wStar*wStar); 
+      uc = sqrt(uStar*uStar+vStar*vStar); 
```

Comment:
@adimako this was generating nan's becase wStar was indexing one past the end of the velocity array.